docs/source/usage/parameters.rst

```diff
@@ -53,10 +53,13 @@ Setting up the field mesh
 * ``geometry.dynamic_size`` (``boolean``) optional (default: ``true`` for dynamic)
     Use dynamic (``true``) resizing of the field mesh, via ``geometry.prob_relative``, or static sizing (``false``), via ``geometry.prob_lo``/``geometry.prob_hi``.
 
-* ``geometry.prob_relative`` (``float``, unitless) optional (default: ``0.1``)
+* ``geometry.prob_relative`` (``float``, unitless) optional (default: ``3.0``)
     By default, we dynamically extract the minimum and maximum of the particle positions in the beam.
     The field mesh is expanded beyond the physical extent of particles by this factor.
```

Comment:
todo: explain that 0.1 means 10% larger.
people could misunderstand this as 1.1 is 10% larger